WordPress/Classes/Services/AccountSettingsService.swift

```diff
@@ -151,6 +151,13 @@ class AccountSettingsService {
         return notificationObserver.map(getSettings).startWith(getSettings())
     }
 
+    func primarySiteNameForSettings(settings: AccountSettings?) -> String {
+        let service = BlogService(managedObjectContext: context)
+        let blog = service.blogByBlogId(settings?.primarySiteID)
+        
+        return blog?.settings?.name ?? String()
```

Comment:
Why not return an optional `String?` instead of an empty one. And more of a style preference I guess but I'm curious, why `String()` and not `""`?